install_files/ansible-base/roles/build-securedrop-app-code-deb-pkg/files/usr.sbin.apache2

```diff
@@ -154,6 +154,8 @@
   /var/www/securedrop/i18n.pyc rw,
   /var/www/securedrop/source.py r,
   /var/www/securedrop/source.pyc rw,
+  /var/www/seucredrop/source_app/__init__.py r,
+  /var/www/seucredrop/source_app/__pycache__/** rw,
```

Comment:
I don't see __pycache__ anywhere else in usr.sbin.apache2 therefore I suppose it is not necessary

Reply:
We also don't have modules anywhere else that compile things to pycache it seems, so I think this might be the first time we end up bumping our heads against this. But I'll worry about this later and since I'm focusing on the refactor right now.